packages/preview/dnd/0.0.2/README.md

```diff
@@ -0,0 +1,238 @@
+# Typst Dungeons and Dragons Template
+
+Create fancy-looking homebrewn Dungeons and Dragons 2024 edition content using
+this [typst](https://www.github.com/typst/typst) template.
+
+Check out the example [document](examples/dnd.typ) and the resulting [PDF file](examples/dnd.pdf).
```

Comment:
These two links won't work on Typst Universe, you should provide full links to your GitHub repository.